core/Error.h

```diff
@@ -36,8 +35,11 @@ class ErrorColors {
 
 public:
     ErrorColors() = delete;
-    template <typename... Args> static std::string format(std::string_view msg, const Args &...args) {
-        return fmt::format(replaceAll(msg, coloredPatternSigil, coloredPatternReplace), args...);
+    template <typename... Args> static std::string format(fmt::format_string<Args...> msg, Args &&...args) {
+        fmt::string_view msgFmtStrView = msg;
+        std::string_view inWhat(msgFmtStrView.data(), msgFmtStrView.size());
+        return fmt::vformat(replaceAll(inWhat, coloredPatternSigil, coloredPatternReplace),
+                            fmt::make_format_args(args...));
```

Comment:
I think that this one basically has to use vformat because there's no (easy?) way for us to make `replaceAll` consteval.